docker/src/main/java/clocker/docker/entity/container/DockerContainerImpl.java

```diff
@@ -420,13 +425,20 @@ private DockerTemplateOptions getDockerTemplateOptions() {
         }
         options.env(env);
 
-        // Entrypoint
+        // Entrypoint and commands
+        // TODO parse string into list?
```

Comment:
Yes, probably good to support coercion of `String` to a `List`, but would need to understand shell quoting, since splitting on spaces is not going to work with e.g. `echo "Hello, World!"` (returns three tokens)